### PR TITLE
Move

### PR DESCRIPTION
**Pull request by UTDQi**
Added Map and Control

### CODE REVIEW

**Comment by stalight** [comment]
Also delete build directory because those cause conflicts (I think)

**Comment by stalight** [comment]
Also do some unit tests to make sure it works

File: src/main/java/usecase_playeractions/Map.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+     *, then place the player on that tile
+     *, triggers the event on that tile, and return True.
+     * @return true if the move is successful, false otherwise
+     */
+    public boolean move(Player p,int x, int y){
+        int[] location = p.getPlayerLocation();
+
+        if(onBoard(location[0]+x,location[1]+y)){
+            return moveTo(p,location[0]+x,location[1]+y);
+        }else{
```
Comment (HongyiZhan) [inline src/main/java/usecase_playeractions/Map.java:67]: beautiful

**Review by HongyiZhan — APPROVED** [review]
good

**Review by jchenDDD — APPROVED** [review]

**Pull request merged**